### DISCUSSION TITLE
How to change language dynamically

### DISCUSSION THREAD

**Discussion by ydeera** [GrapesJS/grapesjs#5247] [Q&A]
Hi, I'm working with Vue3 + grapsjs@0.21.3, and I tried to rerender labels instantly when users change language👇
![watch](https://github.com/GrapesJS/grapesjs/assets/52625550/13b87995-f55c-430a-90b6-75047fc7aa2c)
but it only works after refreshing, and the render() method looks weird
![render](https://github.com/GrapesJS/grapesjs/assets/52625550/a84a1f24-1749-42cf-927e-668c920d0c53)
Do I use it wrong? Or you have any solutions to fix this? Thanks in advance( •̀ ω •́ )✧

**Comment by artf** [comment]
Yeah at the moment the language update requires you to rerender the editor 😞

**Reply by ydeera** [reply]
Thanks.  Are there other methods to rerender the editor? As you can see, I use the ` render` method, then blocks just disappeared😰

**Reply by artf** [reply]
Yeah unfortunately the render method wasn't meant to be called multiple times here so I believe some element references are not properly cleared post-rerender. 
Probably right now the only way to update the default UI with a new language is to reinitialize the editor.
